system/security/plugins/jwtSecurity/jwtSecurity.cpp

```diff
@@ -308,7 +308,8 @@ class CJwtSecurityManager : implements IDaliLdapConnection, public CBaseSecurity
             // Create the JWT token verifier with known claims
             auto jwtVerifier = jwt::verify()
                                 .with_subject(subject)
-                                .with_audience(clientID);
+                                .with_audience(clientID)
+                                .leeway(2);
```

Comment:
Are you confident in the 2 seconds value, or should this be adjustable in config?

Reply:
I am confident.  If there are examples of this not working correctly then I think the fault would lie in the environment (e.g. clocks not synchronized properly) rather than in code.